src/components/grid/row.js

```diff
@@ -10,58 +9,88 @@ const Row = styled.div`
   flex-wrap: wrap;
   
   ${p => css`
-    ${DIMENSIONS.map(d =>
-    config(p).container[ d ] && config(p).media[ d ]`
+    ${DIMENSIONS.map(
+      d =>
+        config(p).container[d] &&
+        config(p).media[d]`
       margin-left: -${config(p).gutterWidth[d] / 2}rem;
       margin-right: -${config(p).gutterWidth[d] / 2}rem;
-    `)}
+    `,
+    )}
   `}
 
-  ${p => p.reverse && css`
-    ${Array.isArray(p.reverse)
-    ? DIMENSIONS.map(d =>
-      config(p).breakpoints[d] && config(p).media[d]`
-        flex-direction: ${p.reverse.indexOf(d) !== -1 ? `row-reverse` : `row`};
-      `)
-    : 'flex-direction: row-reverse;'}
-  `}
+  ${p =>
+    p.reverse &&
+    css`
+      ${Array.isArray(p.reverse)
+        ? DIMENSIONS.map(
+            d =>
+              config(p).breakpoints[d] &&
+              config(p).media[d]`
+         ${
+           p.reverse.indexOf(d) !== -1
+             ? `
+             flex-direction: row-reverse;
+             flex-wrap: wrap-reverse;
+             `
+             : `
+                 flex-direction: row;
```

Comment:
@prxg22  , I'm sorry, but this time I missed something. Here we have to define the `flex-wrap: wrap`, otherwise the larger breakpoints will inherit the wrap-reverse. Making this change, we have to change some lines in the tests as well. I think with that, we can merge.

```suggestion
                 flex-direction: row;
                 flex-wrap: wrap;
```

Reply:
@santosfrancisco  done :+1: